flexmeasures_client.py

```diff
@@ -176,6 +176,16 @@ def get_schedule(self, kwargs, **fnc_kwargs):
             headers={"Authorization": self.fm_token},
         )
         self.check_deprecation_and_sunset(url, res)
+        if res.status_code == 303):
+            new_url = res.headers.get("location")
```

Comment:
Does the returned location include a (new) schedule_id or is this a generic fallback URL?

Reply:
Yes, the location includes the new schedule id.

Reply:
@victorgarcia98 For API clients, does the location return the full url or only the part with `/api/v3_0/sensors/<id>/schedules/<uuid>`, as we test [here](https://github.com/FlexMeasures/flexmeasures/blob/3a9d52430973bf6fcc6b8c604aba00b8baf7d3aa/flexmeasures/api/v3_0/tests/test_sensor_schedules.py#L443-L447)? The location url is created [here](https://github.com/FlexMeasures/flexmeasures/blob/3a9d52430973bf6fcc6b8c604aba00b8baf7d3aa/flexmeasures/api/v3_0/sensors.py#L475-L481) using `flask.url_for`.

Reply:
From testing it only returns a partial url

Reply:
https://github.com/FlexMeasures/flexmeasures/pull/888 will make sure the server returns the full url instead, thanks @GustaafL!